docs/source/getting-started/installation.rst

```diff
@@ -7,47 +7,35 @@ Installing the SDK
 1. Install the dependencies
 ***************************
 
-The SDK currently runs on Windows, MacOS or Linux.
+The SDK currently runs on Windows, macOS and Linux.
 
 You need to install:
 
 1. `Visual Studio Code <https://code.visualstudio.com/download>`_.
-2. JDK 8 or greater.
+2. JDK 8 or greater. Any JDK will do, but if you don't have a preference, here is a link to `AdoptOpenJDK <https://adoptopenjdk.net>`_.
 
-   You can get the JDK from `Zulu 8 JDK <https://www.azul.com/downloads/zulu/>`_ or `Oracle 8 JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_ (requires you to accept Oracle's license).
-   As part of the installation process you might need to set up the ``JAVA_HOME`` variable. You can find here the instructions on how to do it on :doc:`Windows, Mac OS, and Linux <path-variables>`.
-
-.. toctree::
-   :hidden:
-
-   path-variables
+   As part of the installation process you might need to set up the ``JAVA_HOME`` variable. You can find here the instructions on how to do it on :doc:`Windows,macOS, and Linux <path-variables>`.
 
 2. Install the SDK
 *******************
 
-In order to install the DAML SDK you will first need to
+Windows 10
+==========
 
-1. download and install it and
-2. **add** ``~/.daml/bin`` **to your** ``PATH`` **variable settings**.
-   You need to do this in order to be able to call :doc:`daml assistant's commands </tools/assistant>` with ease.
-   This will allow you easier workflow when developing your DAML application.
+Download and run the installer_.
```

Comment:
Worth pointing out that this adds `daml` to PATH automatically just to avoid people getting confused by the following sections for mac and linux?

Reply:
I don't think most Windows users worry about their PATH, nor read the install instructions for Linux/macOS. But I may be wrong; I'll add a sentence to that effect.

Reply:
We did have a user on the forum that asked exactly about this. That said, this was before we reordered the sections so not sure if this is still an issue.